Formula/gitui.rb

```diff
@@ -22,7 +23,7 @@ def install
 
   test do
     system "git", "clone", "https://github.com/extrawurst/gitui.git"
-    (testpath/"gitui").cd { system "git", "checkout", "v0.7.0" }
+    (testpath/"gitui").cd { system "git", "checkout", "v0.8.0" }
```

Comment:
Please use
```suggestion
    (testpath/"gitui").cd { system "git", "checkout", "v#{version}" }
```

Reply:
After consulting with @vladimyr  (the original author of the formulae) over at https://github.com/extrawurst/gitui/pull/137 I tend to follow his strategy and not update the version on each release unless a breaking change requires it.